cmd/mesosturbo.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	goflag "flag"
+	"runtime"
+	"github.com/turbonomic/mesosturbo/cmd/service"
+	"github.com/golang/glog"
+	"github.com/spf13/pflag"
```

Comment:
Package pflag is a drop-in replacement for Go's flag package.
Is the goflag still in use?